new-dti-website/components/blob.tsx

```diff
@@ -1,6 +1,24 @@
+/**
+ * `RedBlob` Component - Renders a stylized red blob element with customizable position and intensity.
+ *
+ * @remarks
+ * This component renders a large, rounded red blob with a radial gradient, which can be positioned
+ * dynamically based on the `className` prop and have varying intensity (opacity) based on the `intensity` prop.
+ *
+ * @safelist
+ * To ensure TailwindCSS handles dynamically generated position classes like `left-[-150px]` or `top-[-96px]`,
+ * safelisting has been added in the `tailwind.config.js` file. This prevents Tailwind from purging these
+ * classes during the production build, ensuring the dynamic class names are retained and applied correctly.
+ * The safelist uses regular expressions to cover common classes for positioning (`left`, `right`, `top`, `bottom`)
+ * with negative values as well (e.g., `-top-96`, `-left-150px`).
+ *
+ * @param props - Contains:
+ *   - `className`: The TailwindCSS class to adjust the positioning and styling of the blob.
+ *   - `intensity`: A numeric value between 0.1 and 1.0 that adjusts the opacity of the blob.
+ */
 const RedBlob: React.FC<{ className?: string; intensity: number }> = ({ className, intensity }) => (
   <div
-    className={`absolute h-[600px] w-[600px] rounded-full z-0 ${className}`}
+    className={`absolute h-[600px] w-[600px] rounded-full -z-10 ${className}`}
```

Comment:
This change seemed to change the blobs on other pages. We can fix this in a later PR when we fix all the styling, but just to point it out.

Reply:
Hmm ok I'll try to increase the z-index of the text itself since I saw that the blob was ontop of it before for the pages.  I'll look more into this before!

Reply:
I think not just the text, but the blob is being covered by the black background too.
<img width="1209" alt="Screenshot 2024-10-27 at 6 32 59 PM" src="https://github.com/user-attachments/assets/fb4fa0fa-ba25-4ade-800e-0763c87b4b07">